cosmoz-suggestions.js

```diff
@@ -0,0 +1,88 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import {
+	component, useCallback
+} from 'haunted';
+
+import { scroll } from 'lit-virtualizer';
+import '@polymer/paper-ripple';
+import '@polymer/paper-material';
+import '@polymer/paper-item';
+import { useSuggestions } from './lib/hooks/use-suggestions';
+
+const Suggestions = ({
+	items, onSelect
+}) => {
+	const {
+			index,
+			rangechange,
+			scrollToIndex,
+			mouseenter,
+			click
+		} = useSuggestions({
+			items,
+			onSelect
+		}),
+		renderItem = useCallback(
+			(item, i) => html`
+				<paper-item
+					role="option"
+					data-index=${i}
+					@mouseenter=${mouseenter}
+					@click=${click}
+					@mousedown=${e => e.preventDefault()}
+				>
+					<div>${item.text}</div>
+					<paper-ripple></paper-ripple>
+				</paper-item>
+			`,
+			[mouseenter]
```

Comment:
```suggestion
			[mouseenter, click]
```

Reply:
Thanks but this is going to change either way. Needs to be `highlight` and `select` instead of the very specific `mouseenter` and `click` names